src/app/ssmt/operations/operations.component.html

```diff
@@ -9,5 +9,58 @@ <h3>Operating Conditions</h3>
       <app-operating-costs [(form)]="operationsForm" [selected]="selected" [settings]="settings" (emitSave)="save()"
         [inSetup]="inSetup" [idString]="idString" [isBaseline]="isBaseline"></app-operating-costs>
     </div>
+
+  <div class="pt-2 ml-2 mr-2">
+    
+  <label class="group-label w-100">Carbon Emissions From Fuel</label>
+  <div class="form-group">
+    <label for="{{'energyType'+idString}}">Energy Type</label>
```

Comment:
Not sure the "Carbon Emissions From Fuel" portion is correct. If "Electricity" is selected for "Energy Type" all the fuel selections are still there.